### PR TITLE
Simple Gitpod support

### PR DESCRIPTION
**Pull request by danielSanchezQ**
Gitpod has some free tier for opensource projects. It can be helpful as a backup dev environment.

A live demo can be accessed from the pulling branch at: https://gitpod.io/#https://github.com/danielsanchezq/jormungandr/tree/gitpod

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+vscode:
+  extensions:
+    - matklad.rust-analyzer
+    - serayuzgur.crates
+    - Swellaby.vscode-rust-test-adapter
+    - hbenl.vscode-test-explorer
```
Comment (mzabaluev) [inline .gitpod.yml:6]: What do these extensions do?

**Review by mzabaluev — COMMENTED** [review]
Cool!

File: .gitpod.yml (hunk 1 above)
Comment (danielSanchezQ) [reply .gitpod.yml:6]: They install the necesary tools into VScode:
    - serayuzgur.crates:  Autocomplete cargo crates in the toml
    - Swellaby.vscode-rust-test-adapter: Run tests from the gui.
    - hbenl.vscode-test-explorer: test report in the gui too.

**Comment by minikin** [comment]
Closing it. Feel free to reopen if we really need it.

**Pull request closed without merging**